build.gradle

```diff
@@ -1,106 +1,23 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
 buildscript {
   repositories {
-    mavenCentral()
-    mavenLocal()
-    maven {
-      url 'https://plugins.gradle.org/m2/'
-    }
+    jcenter()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:1.2.3'
-    classpath 'com.github.dcendents:android-maven-plugin:1.2'
-    classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
-    classpath 'net.researchgate:gradle-release:2.4.0'
-  }
-}
-
-apply plugin: 'com.android.library'
-apply plugin: 'com.github.dcendents.android-maven'
-apply plugin: 'com.neenbedankt.android-apt'
-apply plugin: 'checkstyle'
-apply plugin: 'net.researchgate.release'
-
-group = GROUP
-version = VERSION_NAME
-
-release {
-  versionProperties = ['VERSION_NAME']
-}
-
-afterReleaseBuild.dependsOn uploadArchives
-
-defaultTasks = ['clean', 'assembleDebug', 'checkstyle', 'copyTask', 'testDebug', 'install']
```

Comment:
Default tasks should be declared in root `build.gradle`. As a result tests are not currently running on Circle CI.